package.json

```diff
@@ -67,7 +67,8 @@
     "prettier": "prettier --write \"{packages,services}/**/*.{js,ts,tsx,json}\"",
     "prettier-check": "prettier --check \"{packages,services}/**/*.{js,ts,tsx,json}\"",
     "e2e:chromium": "node -r dotenv/config node_modules/.bin/testcafe chromium ./tests/e2e.ts -s takeOnFails=true --assertion-timeout 10000",
-    "e2e:firefox": "node -r dotenv/config node_modules/.bin/testcafe firefox ./tests/e2e.ts -s takeOnFails=true --assertion-timeout 10000"
+    "e2e:firefox": "node -r dotenv/config node_modules/.bin/testcafe firefox ./tests/e2e.ts -s takeOnFails=true --assertion-timeout 10000",
+    "prepare": "prisma generate"
```

Comment:
Generates the typings for Prisma

Reply:
Good points! This will be followed by a new PR pointed to the same feature branch.